### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Spark Submission_Screen Reader: Narrator/NVDA is not reading the label name for the dropdown as ‘Cluster name’

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3559]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.3.1; #ScreenReader; #C+E-H; #JavatoolingforIntelliJ/Eclipse; #Narrator; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator, NVDA
MAS Violated: MAS1.3.1

**Repro Steps:** 
1) Start Narrator/NVDA. Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
6) Tab till Next button and press enter. Press Finish button
7) In Project Explorer, Expand the created Scala project right-click src, New-->Other and press enter
8) Tab till Scala Wizards, expand and select 'Scala Object' and press next
9) Enter the name of the file and press Finish button
10) Open Scala file and write any code in the editor
11) In Project Explorer view, right-click your project, choose Azure-->Submit Spark Application to HDInsight
12) Tab till ‘Cluster Name’ dropdown and verify if screen reader is reading the name of the label when the focus is on the dropdown
 
**Actual Result:** 
Narrator/NVDA is not reading the label name for the dropdown as ‘Cluster name’

**Expected Result:** 
When focus is on the cluster dropdown Narrator/NVDA should read the label name as ‘Cluster name’

**User Impact:** 
Screen Reader users will no know what to be selected in the dropdown if screen reader does not read the label name

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_5_Narrator is not reading teh label name for Cluster name.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635530/Bug_5_Narrator.is.not.reading.teh.label.name.for.Cluster.name.pptx)

**Comment by t-rufang** [comment]
Note: Screen reader is not the text "zhouytestgen1" on the combobox.
![image](https://user-images.githubusercontent.com/32627233/65827915-d3207600-e2c7-11e9-9b3c-31d5c1211f0e.png)

**Comment by t-rufang** [comment]
Fixed in #3731

**Comment by Tanushagottipati** [comment]
Regressed in the below version and the issue is fixed. Hence closing the bug and attached latest screenshot for reference

Environment Details:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.19.0

![Screenshot (652)](https://user-images.githubusercontent.com/41109135/69145639-1fcc4480-0af4-11ea-8288-003516ab7634.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749670
